### ISSUE TITLE
Incorrect Album Titles When Listening to YouTube Music on Shuffle

### ISSUE THREAD

**Issue by leemander** [kawaiiDango/pano-scrobbler#466]
**Describe the bug**
When shuffle playing a playlist containing music from a variety of different albums on the YouTube Music app often the previous track's album title will be submitted incorrectly for the currently playing track. 

**Steps to Reproduce**
Shuffle play a playlist containing music from various albums.  

**App version**
3.27

**Additional info**

**Comment by everypizza1** [comment]
I've experienced this same bug when playing local files from Spotify, but only local files without album art in the metadata.

**Comment by leemander** [comment]
> I've experienced this same bug when playing local files from Spotify, but only local files without album art in the metadata.

Could be an issue with YouTube's metadata then perhaps.

**Comment by kawaiiDango** [comment]
This is a bug in YouTube music and this issue happens with all scrobblers, the last time I checked. I don't know why this happens with Spotify too, will check some time

**Comment by replicaacliper** [comment]
So is there no way to fix this? I'm having the same issue with YouTube Music Vanced on my Android phone.

**Comment by replicaacliper** [comment]
Yes I'm still having this issue. One way to replicate it is to play a user-uploaded track (with no album metadata, [e.g.](https://music.youtube.com/watch?v=BPJpMSRR5w8&si=klmPMygTg41M-woj) ) after an automated-uploaded track (with album metadata, [e.g.](https://music.youtube.com/watch?v=BPJpMSRR5w8&si=klmPMygTg41M-woj)). I was just able to replicate this issue on my Android device using these two tracks.

**Comment by DangoSnail** [comment]
Interesting, I've tried replicating it with some songs of my own as well on both my phone and on Bluestacks but I'm unable to replicate it. I even tried linking a different last.fm account as well. I'm not sure why I've stopped getting the bug but it must have been recent because I was still getting it just last month.
